### PR TITLE
Doc v2 s

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/core/regular/saturate.hpp [hunk 1]
```diff
@@ -57,8 +57,7 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!   {   template<value U, scalar_value T>
-//!       U operator()(U x, as<T> t) noexcept;
+//!   {   template<value U, scalar_value T> U operator()(U x, as<T> t) noexcept;
```
Comment (jfalcou) [inline include/eve/module/core/regular/saturate.hpp:60]: missing carriage return

File: include/eve/module/core/regular/sign_alternate.hpp [hunk 2]
```diff
@@ -46,31 +46,28 @@ namespace eve
...
 //!
 //!     * `n` :  [argument](@ref eve::value). Must be signed integral or flint.
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!    **Return value**
 //!
-//!      \f$(-1)^n\f$ is returned.
-//!
+//!      1. \f$(-1)^n\f$ is returned.
```
Comment (jfalcou) [inline include/eve/module/core/regular/sign_alternate.hpp:66]: this looks like a scalar value, try :

```
1. A value containing \f$(-1)^i\f$ in lane \f$i\f$ is returned.
```

File: include/eve/module/core/regular/signnz.hpp [hunk 3]
```diff
@@ -44,42 +44,38 @@ namespace eve
...
-//!      * Computes  [elementwise](@ref glossary_elementwise) the never zero sign of `x`.
-//!
-//!      * For [real](@ref eve::value) `x`,  the call is semantically equivalent to:
-//!        * If x is [positive](@ref eve::is_positive), 1 is returned.
-//!        * If x is [negative](@ref eve::is_negative), -1 is returned.
+//!      1. Computes  [elementwise](@ref glossary_elementwise) the never zero sign of `x`.
+//!          - If x is [positive](@ref eve::is_positive), 1 is returned.
+//!          - If x is [negative](@ref eve::is_negative), -1 is returned.
+//!      2. [The operation is performed conditionnaly](@ref conditional).
+//!      3. `eve;::signnz[pedantic](x)` returns nan for nan input
```
Comment (jfalcou) [inline include/eve/module/core/regular/signnz.hpp:71]: 3. Same as 1. but returns nan for nan input

File: include/eve/module/core/regular/sqrt.hpp [hunk 4]
```diff
@@ -40,36 +40,35 @@ namespace eve
...
-//!      T sqrt(T x) noexcept;
+//!      // Regular overload
+//!      constexpr auto sqrt(value auto x)                          noexcept; // 1
+//!
+//!      // Lanes masking
+//!      constexpr auto sqrt[conditional_expr auto c](value auto x) noexcept; // 2
+//!      constexpr auto sqrt[logical_value auto m](value auto x)    noexcept; // 2
+//!
+//!      // Semantic options
+//!      constexpr auto abs[raw](value auto x)               noexcept; // 3
```
Comment (jfalcou) [inline include/eve/module/core/regular/sqrt.hpp:51]: should be sqrt

File: include/eve/module/core/regular/sum_of_prod.hpp [hunk 5]
```diff
@@ -46,39 +46,37 @@ namespace eve
...
+//!      // Regular overload
+//!      constexpr auto sum_of_prod(floating_value auto x, floating_value auto y,
+//!                                  floating_value auto z, floating_value auto t)                          noexcept; // 1
+//!
+//!      // Lanes masking
+//!      constexpr auto sum_of_prod[conditional_expr auto c](floating_value auto x, floating_value auto y,
+//!                                                           floating_value auto z, floating_value auto t) noexcept; // 2
+//!
+//!      // Semantic exclusive options
+//!      constexpr auto sum_of_prod[raw](floating_value auto x, floating_value auto y,
```
Comment (jfalcou) [inline include/eve/module/core/regular/sum_of_prod.hpp:58]: alignemnt of parameters and position of , is wrong in the whoel block

File: include/eve/module/math/regular/horner.hpp [hunk 6]
```diff
@@ -57,12 +57,12 @@ namespace eve
 //!   namespace eve
 //!   {
 //!      // Regular overloads
-//!      constexpr auto horner(value auto x, value auto ...ci)                         noexcept; // 1
+//!      constexpr auto horner(value auto x, value auto ci...)                         noexcept; // 1
```
Comment (jfalcou) [inline include/eve/module/math/regular/horner.hpp:60]: This commit just revert all the fixes from before, be careful.

File: include/eve/module/math/regular/sinhcosh.hpp [hunk 7]
```diff
@@ -52,24 +52,29 @@ namespace eve
...
+//!      * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!      * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!   }
 //!
 //! **Return value**
 //!
-//! Returns the [elementwise](@ref glossary_elementwise) pair of values containing the
-//! hyperbolic sine and the hyperbolic cosine of the input.
+//!   1 .The computation returns a tuple-like whose elements are `sinh(x)` and `cosh(x)`
```
Comment (jfalcou) [inline include/eve/module/math/regular/sinhcosh.hpp:74]: sin and cos no ?

File: include/eve/module/math/regular/sinpic.hpp [hunk 8]
```diff
@@ -42,32 +42,32 @@ namespace eve
...
 //!
-//! Returns the [elementwise](@ref glossary_elementwise) sine of the input times \f$\pi\f$  divided
-//! by the input times \f$\pi\f$.
-//!
-//! In particular:
-//!
-//!   * If the element is \f$\pm0\f$, \f$1\f$ is returned.
-//!   * If the element is \f$\pm\infty\f$, 0 is returned.
-//!   * If the element is a `Nan`, `NaN` is returned.
+//!    1. Returns the [elementwise](@ref glossary_elementwise) sine of the input times \f$\pi\f$  divided
```
Comment (jfalcou) [inline include/eve/module/math/regular/sinpic.hpp:62]: the sine or the normalized cardinal sine ?

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/core/regular/saturate.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/saturate.hpp:60]: done

File: include/eve/module/math/regular/horner.hpp (hunk 6 above)
Comment (jtlap) [reply include/eve/module/math/regular/horner.hpp:60]: done

File: include/eve/module/math/regular/sinhcosh.hpp (hunk 7 above)
Comment (jtlap) [reply include/eve/module/math/regular/sinhcosh.hpp:74]: no

File: include/eve/module/math/regular/sinpic.hpp (hunk 8 above)
Comment (jtlap) [reply include/eve/module/math/regular/sinpic.hpp:62]: the text is correct

**Review by jfalcou — APPROVED** [review]

**Pull request merged**